rel/env.sh.eex

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+# Set the release to work across nodes.
+# RELEASE_DISTRIBUTION must be "sname" (local), "name" (distributed) or "none".
+export RELEASE_DISTRIBUTION=name
+if ! RELEASE_NODE=oli@$(curl -s http://169.254.169.254/latest/meta-data/local-ipv4)
```

Comment:
Didn't follow you here, do you mean extract what we get with `(curl -s http://169.254.169.254/latest/meta-data/local-ipv4)` to a variable? Or just the AWS IP? It will always be the same for retrieving instance metadata: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instancedata-data-retrieval.html

Reply:
Disregard - didn't realize it was local IP to instance. Thanks.